### ISSUE TITLE
[Hueman] Font awesome => upgrade to FA5

### ISSUE THREAD

**Issue by ghost** [AlxMedia/hueman#617]
see https://github.com/presscustomizr/customizr/issues/1364

**Comment by eri-trabiccolo** [comment]
Design decisions to take regarding to some icons:
As you know basically all the former fa icons are now rendered with the Solid icons, which are bold.
In Hueman they're vastly used, e.g. the angular double-arrow and the arrow angular arrow for:
1) topbar-toggle-down
2) sidebar-toggles
3) back-to-top

the new solid icons, aren't that "nice".
E.g. fa4 sidebar-toggle:
<img width="26" alt="old-fa" src="https://user-images.githubusercontent.com/7689242/35386975-4c152e48-01ce-11e8-8eb2-ba3d18bfec20.png">

fa5 sidebar-toggle (using the equivalent icon):
<img width="25" alt="new" src="https://user-images.githubusercontent.com/7689242/35386981-501abbc0-01ce-11e8-98ad-ee593b20db04.png">

If we want to keep the same icons we can slighlty decrease the font-size so that they don't appear that ugly.
Otherwise we need replacements.
What's your opinion?

**Comment by eri-trabiccolo** [comment]
here you can find the first fix release: https://github.com/presscustomizr/hueman/pull/627
If you agree, I would like to "group" the font-awesome icons CSS used in the main theme files by different selectors.
There are a lot of repetition at the moment (there already were).
E.g. 
From the current Hueman stabele version: We say here to use the font awesome font-family:
https://github.com/presscustomizr/hueman/blob/v3.3.25/assets/front/css/_parts/0_1_base_styles.css#L184
and we say it here too:
https://github.com/presscustomizr/hueman/blob/v3.3.25/assets/front/css/_parts/0_3_common_elements.css#L87

and so on.

I want to group these selectors in a single place so to have a smaller (even minified) CSS.
Basically the same thing we do for Customizr here:
https://github.com/presscustomizr/customizr/blob/v4.0.16/inc/assets/less/tc_custom.less#L100

This group will be put here:
https://github.com/presscustomizr/hueman/blob/v3.3.25/assets/front/css/_parts/0_1_base_styles.css#L39

**Comment by eri-trabiccolo** [comment]
pro/addons: https://github.com/presscustomizr/hueman-pro-addons/pull/126
skope: https://github.com/presscustomizr/skope/pull/6
hueman: https://github.com/presscustomizr/hueman/pull/627

I didn't do the "grouping selectors" part I mentioned in https://github.com/presscustomizr/hueman/issues/617#issuecomment-360752276
as that's not urgent even if it will decrease the CSS size, but you might prefer to keep things separated as they are now.

**Comment by ghost** [comment]
Good point about grouping those rules.
But it is not critical, so yes this can wait I agree too.
